mava/evaluator.py

```diff
@@ -262,7 +277,7 @@ def evaluator_setup(
     """Initialise evaluator_fn."""
     # Get available TPU cores.
     n_devices = len(jax.devices())
-
+    win_rate = config["env"]["env_name"] == "HeuristicEnemySMAX"
```

Comment:
Slightly widens this condition if we use the other envs in the future

```suggestion
    win_rate = isinstance(env, (SMAX, HeuristicEnemySMAX, LearnedPolicyEnemySMAX))
```

Reply:
The issue here is that env is not an instance of SMAX... as it is a wrapped env therefore this `isinstance` will be very nested by mentioning `env._env._env...` depending on how many wrappers are added